php/spec/module-phar_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'spec_helper.rb'
+
+describe 'php::module-phar' do
+
+  let(:runner)    { ChefSpec::Runner.new }
+  let(:chef_run)  { runner.converge(described_recipe) }
+  let(:node)      { runner.node }
+
+  it 'adds ppa mirror configuration' do
+    expect(chef_run).to include_recipe('apt::easybib')
```

Comment:
@baccenfutter @fh Before we add more of these, maybe we do something like:

```
expect(chef_run).to include_recipe('php::dependencies-ppa')
```

Instead? And then we add a test on `php::dependencies-ppa` including what we think it must?

Something to consider for the future. :) Doesn't have to be decided now, but I thought I'd raise it.